doc/regression-tests.md

```diff
@@ -53,6 +53,12 @@ You *will* need to update it if you:
   * change the "questions & responses" file
   * change the routing logic
 
+Example:
+
+```shell
+bundle exec rails r script/generate-responses-and-expected-results-for-smart-answer.rb marriage-abroad
```

Comment:
Nope not too late. I can either change the description of the example to say `Example, to run against Marriage Abroad` as it's our most common use case or change the example to have `<smart-answer>`. I'd prefer the former, but what do you think @ikennaokpala?